src/main/java/com/stripe/model/issuing/Authorization.java

```diff
@@ -1,206 +1,343 @@
-   */
-  public static Authorization retrieve(String id, RequestOptions options) throws StripeException {
-    return retrieve(id, null, options);
+  /** Approves a pending Issuing <code>Authorization</code> object. */
+  public Authorization approve(RequestOptions options) throws StripeException {
+    return approve((Map<String, Object>) null, options);
   }
 
-  /**
-   * Retrieve an issuing authorization.
-   */
-  public static Authorization retrieve(String id, Map<String, Object> params,
-      RequestOptions options) throws StripeException {
-    return request(RequestMethod.GET, instanceUrl(Authorization.class, id), params,
-      Authorization.class, options);
+  /** Approves a pending Issuing <code>Authorization</code> object. */
+  public Authorization approve(Map<String, Object> params) throws StripeException {
+    return approve(params, (RequestOptions) null);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="update">
-  /**
-   * Update an issuing authorization.
-   */
-  @Override
-  public Authorization update(Map<String, Object> params) throws StripeException {
-    return update(params, null);
+  /** Approves a pending Issuing <code>Authorization</code> object. */
+  public Authorization approve(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url =
+        String.format(
+            "%s%s",
+            Stripe.getApiBase(),
+            String.format("/v1/issuing/authorizations/%s/approve", this.getId()));
+    return request(ApiResource.RequestMethod.POST, url, params, Authorization.class, options);
   }
 
-  /**
-   * Update an issuing authorization.
-   */
-  @Override
-  public Authorization update(Map<String, Object> params, RequestOptions options)
+  /** Declines a pending Issuing <code>Authorization</code> object. */
+  public Authorization decline() throws StripeException {
+    return decline((Map<String, Object>) null, (RequestOptions) null);
+  }
+
+  /** Declines a pending Issuing <code>Authorization</code> object. */
+  public Authorization decline(RequestOptions options) throws StripeException {
+    return decline((Map<String, Object>) null, options);
+  }
+
+  /** Declines a pending Issuing <code>Authorization</code> object. */
+  public Authorization decline(Map<String, Object> params) throws StripeException {
+    return decline(params, (RequestOptions) null);
+  }
+
+  /** Declines a pending Issuing <code>Authorization</code> object. */
+  public Authorization decline(Map<String, Object> params, RequestOptions options)
       throws StripeException {
-    return request(RequestMethod.POST, instanceUrl(Authorization.class, this.id), params,
-      Authorization.class, options);
+    String url =
+        String.format(
+            "%s%s",
+            Stripe.getApiBase(),
+            String.format("/v1/issuing/authorizations/%s/decline", this.getId()));
+    return request(ApiResource.RequestMethod.POST, url, params, Authorization.class, options);
```

Comment:
13) Update (custom): Instance method to update resource having business-logic impact besides mutating the data. Same behavior of relying on instance ID attribute as the standard update method.

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/issuing/Authorization.java#L269-L278